### ISSUE TITLE
New feature and feedback label

### ISSUE THREAD

**Issue by abdulloooh** [teezzan/An-Ayah-A-Day#14]
Can we have a label for new feature and feedback hanging somewhere right or left?

New feature allows users to know new features we have included and how to use them

Feedback allows us to get feed backs from users

**Comment by teezzan** [comment]
Its a good one. Check out vue-bootstrap side bar. Cool feature to be used

**Comment by abdulloooh** [comment]
Alright, I'll give it a go.
It's awesome if we link it to a google doc and a google form for new features details and feedback right? 
Since no dynamism from Backend that can handle dynamic updates for us

**Comment by abdulloooh** [comment]
You agree on this?

**Comment by teezzan** [comment]
Yup. Vue-bootstrap side bar is the best for such implementation.
